app/components/requests-item.js

```diff
@@ -3,27 +3,32 @@ import { computed } from 'ember-decorators/object';
 
 export default Ember.Component.extend({
   classNames: ['request-item'],
-  classNameBindings: ['requestClass'],
+  classNameBindings: ['requestClass', 'highlightedClass'],
   tagName: 'li',
 
   @computed('request.message')
   isGHPages(message) {
     return message === 'github pages branch';
   },
 
-  @computed('request.isAccepted')
-  requestClass(isAccepted) {
-    return isAccepted ? 'accepted' : 'rejected';
+  @computed('request.result')
+  requestClass(result) {
+    return result;
```

Comment:
This change means this be expressed more simply as an alias: `@alias requestClass: null`